opportunity_job_app/src/components/JobOfferDetails/JobOfferDetails.tsx

```diff
@@ -37,6 +38,40 @@ export default function JobOffersDetails({ jobOfferID }: JobOfferDetailsProps) {
         window.location.href = "/job-offers";
     }
 
+    const fetchJobOffers = async () => {
+        try {
+            const newJobOffer = await JobOfferService.findJobOffer(jobOfferID)
+            setJobOffer(newJobOffer)
+        } catch (error) {
+            goBack()
+        }
+    }
+    
+
+    const enrollUser = async () => {
+        if (user) {
+            try {
+                const { account_id } = user;
+                await JobEnrollmentService.addJobEnrollment(jobOfferID, account_id);
+                fetchJobOffers();
+            } catch (error) {
+                console.log("Enrollment error:", error);
+            }
+        }
+    };
+
+    const removeUser = async () => {
```

Comment:
Ova funkcija ne radi ono sto govori da radi (uklanja korisnika).
Najbolji naziv bi bio isti kao i naziv funkcije servisa: `removeJobEnrollment`.
To vazi i za prethodnu `enrollUser` funkciju.